web/src/unread_ops.ts

```diff
@@ -256,31 +258,60 @@ export function mark_as_unread_from_here(
     }
     message_lists.current.prevent_reading();
 
-    // If we have already fully fetched the current view, we can
-    // send the server the set of IDs to update, rather than
-    // updating on the basis of the narrow.
-    let message_ids_to_update;
-    if (message_lists.current.data.fetch_status.has_found_newest()) {
-        message_ids_to_update = message_lists.current
-            .all_messages()
-            .filter(
-                (msg) =>
-                    (include_anchor && msg.id >= message_id) ||
-                    (!include_anchor && msg.id > message_id),
-            )
-            .map((msg) => msg.id);
+    const is_all_fetched = message_lists.current.data.fetch_status.has_found_newest();
```

Comment:
The boolean variable is good, but why not call it `has_found_newest`? `is_all_fetched` can be misleading, since it sounds like the oldest might be fetched too.